core/src/main/java/tech/tablesaw/io/Source.java

```diff
@@ -19,6 +20,11 @@
     protected final Reader reader;
     protected final InputStream inputStream;
     protected final Charset charset;
+    private final Charset UTF8 = Charset.forName("UTF-8");
+
+    public Source(String filename) {
```

Comment:
I'd rather not add so many new constructors to `Source`. If we want to make it easier to call I'd probably do it by adding new methods to `CsvReadOptions`. E.g. where we have `CsvReadOptions.builderFromFile` maybe add a new one with a charset parameter